tools/releases/helm.sh

```diff
@@ -65,7 +65,21 @@ function package {
       find "${dir}/crds" -name "*meshgatewayconfigs.yaml" -delete
     fi
 
-    cr package \
+    # Chart Releaser is always packaging dependencies specified in Chart.yaml
+    # file first. The archives with packaged dependencies lands inside "charts"
+    # directory (i.e. deployments/[...]/charts/[dependency_name]-[version].tgz).
+    # When building the final chart package, it includes inside, the content of
+    # the "charts" directory. When it finds an archive there, it flattens it and
+    # includes in the final package as well. It means if the chart have any
+    # dependencies, the final chart's archive will contain duplicated manifests.
+    # As Chart Releaser will automatically download the dependencies and then
+    # will archive them, we can safely remove the content of "charts" directory
+    # to mitigate the above issue.
+    if [[ -d "${dir}/charts" ]]; then
+      rm -vrf "${dir}/charts/"*
```

Comment:
I'm not quite sure about this way, shouldn't the local charts always have precedence?